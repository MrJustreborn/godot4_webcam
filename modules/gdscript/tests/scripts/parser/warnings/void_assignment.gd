func i_return_void() -> void:
	return


func test():
	var __ = i_return_void()
