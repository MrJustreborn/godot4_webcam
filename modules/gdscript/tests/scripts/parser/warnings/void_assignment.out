GDTEST_OK
>> WARNING
>> Line: 6
>> VOID_ASSIGNMENT
>> Assignment operation, but the function 'i_return_void()' returns void.
